src/shared/nested-dropdown-list/list-item.tsx

```diff
@@ -128,7 +141,11 @@ export const ListItem = <V1, V2, V3>({
         const nextSubItems = item.subItems.values();
 
         return (
-            <List $expanded={item.expanded}>
+            <List
+                $expanded={item.expanded}
+                $level_3={item.keyPath.length + 1 === 3}
```

Comment:
`level_3` doesn't work in the scenario where the first/second level have no subitems

in `ListItemSelector`, try setting the left margin when `$multiSelect` is true

Reply:
please check for this

Reply:
cause horizontal scrollable if we set it margin-left in `ListItemSelector`.